src/rules/radixRule.ts

```diff
@@ -44,20 +44,45 @@ export class Rule extends Lint.Rules.AbstractRule {
     }
 }
 
+function isParseInt(expression: ts.LeftHandSideExpression) {
+    return isIdentifier(expression) && expression.text === "parseInt";
+}
+
+function isPropertyAccessParseInt(
+    expression: ts.LeftHandSideExpression,
+): expression is ts.PropertyAccessExpression {
+    return isPropertyAccessExpression(expression) && expression.name.text === "parseInt";
```

Comment:
this seems like it would ban `myCustomClass.parseInt(arg)`, which could be a perfectly valid use of a non-library function called `parseInt`. true?

please add a test for this case.

Reply:
isPropertyAccessParseInt() will return true for that case yes. However, in conjunction with isPropertyAccessOfProperty() and isPropertyAccessOfIdentifier(), the check is constrained to a specific instances.  A test already exists for this:
`parser.parseInt("123");` on line 6 of the radix rule test.

Reply:
nice